map_view/display-device-location-with-autopan-modes/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
```

Comment:
just remembered that we should update this gradle script as you are doing on master: to remove deprecated features.